src/civic_canister_backend/src/credential.rs

```diff
@@ -313,20 +295,20 @@ async fn remove_credential(
                     credentials.insert(principal, CredentialList(existing_credentials_vec));
                     Ok("Credential removed successfully".to_string())
                 } else {
-                    Err(CredentialError::UnauthorizedSubject(
+                    Err(IssueCredentialError::UnauthorizedSubject(
```

Comment:
Should be RemoveCredentialError and something other than UnauthorizedSubject?

Maybe the enum type should just go back to something more generic like Error or CredentialError if the same values will be used for add/remove/update?

Reply:
Updated